src/vs/editor/browser/viewParts/rulers/rulers.css

```diff
@@ -6,4 +6,5 @@
 .monaco-editor .view-ruler {
 	position: absolute;
 	top: 0;
-}
\ No newline at end of file
+	box-shadow: 1px 0 0 0 var(--vscode-editorRuler-ruler) inset !important;
```

Comment:
Does `!important` need to be used here?

Reply:
I think it doesn't need to be overridden. I remove ``!important``